packages/enzyme-mocha/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "enzyme-mocha",
+  "version": "1.1.0",
+  "description": "Make Mocha and Enzyme play nice",
+  "main": "build/index.js",
+  "repository": "https://github.com/airbnb/enzyme/tree/master/packages/enzyme-mocha",
+  "keywords": [
+    "mocha",
+    "enzyme"
+  ],
+  "author": "Leland Richardson <leland.richardson@airbnb.com>",
+  "license": "MIT",
+  "dependencies": {
+    "jsdomify": "^1.0.2"
+  },
+  "peerDependencies": {
+    "react": "0.13.x || 0.14.x"
```

Comment:
So I was going to add it and realized... it doesn't actually need enzyme.  I'm not sure we want to have a dependency that isn't real?

It begs the question about whether or not these small little utilities are even worth it... but I think we will be able to add more over time.